WordPress/gutenberg
# Navigation: Add missing empty space to print out valid HTML.

## What?

Adding an empty space to have the navigation block producing valid HTML.

Fixes https://github.com/WordPress/gutenberg/issues/59238

## Why?


Because if not, there are two attributes together.

## How?


Adding an empty space.

## Testing Instructions

1. Open a site with TT4.
2. Set up a navigation menu.
3. Visit the front end, view the page source, and confirm that there's an empty space between `data-wp-interactive="core/navigation"` and the following data attribute of the `nav` element.

<img width="505" alt="Screenshot 2024-02-21 at 19 23 45" src="https://github.com/WordPress/gutenberg/assets/252415/f3bb7733-39aa-4170-9a44-31556f794ac5">
